core/arch/arm/kernel/secure_partition.c

```diff
@@ -375,6 +386,146 @@ static TEE_Result sp_open_session(struct sp_session **sess,
 	return TEE_SUCCESS;
 }
 
+static TEE_Result sp_dt_get_u64(const void *fdt, int node, const char *property,
+				uint64_t *value)
+{
+	int len = 0;
+	const fdt64_t *cuint64 = NULL;
```

Comment:
Unnecessary complicated name, where just `p` would suffice. Same below in `sp_dt_get_u32()`

Reply:
Done